exampleCourse/questions/demo/workspace/jupyterlab/question.html

```diff
@@ -1,4 +1,26 @@
 <pl-question-panel>
-  <p>This is a workspace question with an in-browser <a href="https://jupyter.org">JupyterLab</a>.</p>
+  <p>This is a workspace question with an in-browser <a href="https://jupyter.org">JupyterLab</a>.
+  </p>
+
+  <p>
+    The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each
+    number is the sum of the two before it. These are numbered $F_1, F_2, ...$, and
+    \[
+    F_n = F_{n-1} + F_{n-2}
+    \]
+  </p>
+
+  <p>
+    In the notebook, you will be prompted to write a Python function <tt>fib</tt> that takes a number <tt>n</tt>
```

Comment:
```suggestion
    In the notebook <tt>Workbook.ipynb</tt>, you will be prompted to write a Python function <tt>fib</tt> that takes a number <tt>n</tt>
```
Should we be more explicit?